Gemfile.lock

```diff
@@ -862,7 +862,7 @@ DEPENDENCIES
   web-console (= 4.0.4)
 
 RUBY VERSION
-   ruby 2.7.5p203
+   ruby 2.7.1p83
```

Comment:
Could you please revert the `Gemfile.lock` again since we are not doing any changes to gems in this PR?

Otherwise this looks really good now!